encrypted-media-respec.html

```diff
@@ -606,7 +606,83 @@
           <section>
             <h5>Get Supported Configuration</h5>
             <p>Given a <a def-id="keysystems"></a> implementation <var>implementation</var>, <a>MediaKeySystemConfiguration</a> <var>candidate configuration</var>, and <var>origin</var>, this algorithm returns a supported configuration or <code>null</code> as appropriate.</p>
-            <p class="note">Unrecognized dictionary members in <var>candidate configuration</var> are ignored per [[WebIDL]] and will never reach this algorithm. Thus, they cannot be considered as part of the configuration.</p>
+            <p class="note">Unrecognized dictionary members in <var>candidate configuration</var> are ignored per [[WebIDL]] and will never reach this algorithm. Thus, they cannot be considered as part of the configuration.
+            </p>
+            <ol>
+              <li>
+                <dl class="switch">
+                  <dt>
+                    If the <a def-id="option-distinctiveIdentifier"></a> member of
+                    <var>candidate configuration</var> is <a def-id="requirement-optional"></a>:
+                  </dt>
+                  <dd>
+                    <ol>
+                      <li>
+                        <p>
+                          For each <var>distinctive identifier state</var> in the list
+                          [ <a def-id="requirement-optional"></a>, <a def-id="requirement-not-allowed"></a> ]:
```

Comment:
In addition to my comments above, we need to remember that consent was requested for future configurations. If consent was denied for this configuration, we need to reevaluate it (as this PR currently does) AND if this configuration is not supported (this algorithm terminates with `null`), evaluate all remaining configurations with consent denied.

That's actually covered by your update to the Note below. Thus, algorithmically, maybe we are fine. I like that implementations have flexibility to request multiple consents (or equivalent) if necessary. It would be nice if this was more clear, though. Perhaps we can add another Note or move that part of the Note up here or in `requestMediaKeySystemAccess()` to make that clearer to the reader.